settings_test.py

```diff
@@ -82,10 +83,10 @@ def _polite_tmpdir():
 # These are the default languages. If you want a constrainted set for your
 # tests, you should add those in the tests.
 AMO_LANGUAGES = (
```

Comment:
Should we stop constraining the test languages btw? the list is huge now, I don't really see the point.